discord/index.js

```diff
@@ -1,21 +1,15 @@
-// const { Webhook, MessageBuilder } = require('discord-webhook-node');
-// const webhook_url = process.env.WEBHOOK_URL;
-
-// const hook = new Webhook(webhook_url);
-
-// const fireNewReportHook = async (isSubmission, reported_item) => {
-//   const { permalink, author, title, body } = reported_item;
-//   const messageText = isSubmission ? title : body;
-
-//   const embed = new MessageBuilder()
-//     .setTitle(`${isSubmission ? 'Post' : 'Comment'} Reported`)
-//     .setURL(`https://reddit.com${permalink}`)
-//     .addField(author.name, messageText);
-
-//   console.log(embed);
-//   // hook.send(embed);
-// };
-
-// module.exports = {
-//   fireNewReportHook,
-// };
+const { NodeJSBot } = require("nodejsdiscordbot");
+
+const KnicksDiscordBot = class KnicksDiscordBot extends NodeJSBot {
+    constructor(prefix, options = {}) {
+        super(prefix, options);
+        this.once("ready", () => {
+            this.commandCollection.loadCommands(process.cwd() + "/discord/commands");
+            console.log("Discord Bot ready...");
+        })
```

Comment:
What exactly is happening here? On creation of KnicksDiscordBot you fire the "ready" command? 

From here what does this class actually do? 

If there are no other bound methods that can be used, it seems like something that could also become a function

Reply:
"ready" is discord.js confirming that the bot is logged in and working and so then it loads all the commands.
ready isn't fired until the bot is logged in.

The class currently does very little but the aim is to add functions to it when the discord bot needs to do something.